### PR TITLE
Config consolidation

### LINKED ISSUE #2131: Combine cypress config for articles & WS
**Is your feature request related to a problem? Please describe.**
We currently have two sources of config, `cypress/support/config.js` & `cypress/support/worldServices.js`. This makes it v difficult to iterate over all services for generic page tests

**Describe the solution you'd like**
Combine the configs, allowing for the ability to easily iterate over all services and get an example page from each.

**Testing notes**
[Tester to complete]

Dev insight: None, just regression of testing tests

**Additional context**

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Resolves #2131 

**Overall change:** Consolidates all cypress config

**Code changes:**

- Change from three config files in /cypress to one dir with one file for env config and another for service config.
- Document the strict existence of 1 approach to cypress config.
- Revise use of cypress config or app config for translations/string expects - currently, it's mixed between 1. hardcoded 2. in cypress config and 3. app config

---

- [x] I have assigned myself to this PR and the corresponding issues
~~- [ ] Tests added for new features~~
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/integration/cookieBannerWS.js [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
+  const serviceConfig = services[index];
+  const service = index;
 
-// These tests work locally, but fail on Test & Live environments since they have
-// not yet been set up to have the correct translated cookie banners on the error pages
-describeForLocalOnly('World Service Cookie banner Translations', () => {
-  describe('Canonical', () => {
-    services.forEach(serviceName => {
-      it(`should load the relevant translations for ${serviceName}`, () => {
+  // This should be unhacked when this files is consolidated with other cookie testing files.
```
Comment (jamesbhobbs) [inline cypress/integration/cookieBannerWS.js]:
```suggestion
  // This should be unhacked when this file is consolidated with other cookie testing files.
```

**Review by jroebu14 — APPROVED** [review]
LGTM 👍

**Review by dr3 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing, but I will run it by a tester and ensure they've read it

**Comment by jamesbhobbs** [comment]
Run by @jamesbrumpton (whose tests authored are most affected)

**Pull request merged**